web/src/lib/domain-logic/isRemoteWorkerOrSeller.ts

```diff
@@ -0,0 +1,8 @@
+import { ForeignBusinessTypeId } from "@businessnjgovnavigator/shared/";
+import { determineForeignBusinessType } from "@businessnjgovnavigator/shared/domain-logic/determineForeignBusinessType";
+
+export const isRemoteWorkerOrSeller = (foreignBusinessTypeIds: ForeignBusinessTypeId[]): boolean => {
```

Comment:
[pebble] I think this would be better if you passed a business instead of an ID, that way you can check for that it's foreign inside rather than outside.

Reply:
[dust] I also think it might be good to check for undefined state as a guard clause. (And then test that as well) just for robustness sake